### PR TITLE
fix(a380x): Model update main repo changes (cabin lights, door anims)

### PR DESCRIPTION
**Pull request by flogross89**
## Summary of Changes

Changes from intermediate model update
- Cabin lights added
- Door animations added

## Screenshots (if necessary)
none

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/systems.cfg
Comment (2hwk) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/systems.cfg:801]:
```suggestion
LocalVar.76 = A380X_CABIN_HIDDEN
LocalVarDefault.76 = 0
```

Don't think this belongs here

Comment (flogross89) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/systems.cfg]: Was this part of another PR? It's gone now, after merging with master

**Review by 2hwk — APPROVED** [review]

**Comment by 2hwk** [comment]
I've noticed the door animations don't sync with the EFB, I'll be able to fix that up soon, hold off merging until then.

**Pull request merged**
